pallets/did/src/benchmarking.rs

```diff
@@ -296,11 +295,475 @@ benchmarks! {
 
 		let did_call_signature = ecdsa_sign(AUTHENTICATION_KEY_ID, &did_public_auth_key, did_call_op.encode().as_ref()).expect("Failed to create DID signature from raw ecdsa signature.");
 	}: submit_did_call(RawOrigin::Signed(submitter), Box::new(did_call_op), DidSignature::from(did_call_signature))
+
+	/* set_authentication_key extrinsic */
+	set_ed25519_authentication_key {
+		let block_number = T::BlockNumber::zero();
+		let old_did_public_auth_key = get_ed25519_public_authentication_key();
+		let did_subject: DidIdentifierOf<T> = MultiSigner::from(old_did_public_auth_key).into_account().into();
+
+		// fill up public keys to its max size
```

Comment:
I would perhaps add that we are filling up because the # of max public keys = # of max key agreement keys + 3.

Reply:
--> a527303